src/GitVersionTfsTask/task.json

```diff
@@ -35,6 +35,15 @@
       "helpMarkDown": "Whether to update versions in the AssemblyInfo files"
     },
     {
+      "name": "updateAssemblyInfoFilename",
+      "type": "string",
+      "label": "Update Assembly File",
+      "defaultValue": "",
+      "required": false,
+      "helpMarkDown": "Update versions in specified file",
```

Comment:
A little bit more documentation here would be nice: Are wildcards patterns supported, what is the default value if no value is entered (AssemblyInfo.cs or AssemblyInfo.vb)